pkg/certs/cert-inspection/certgraphanalysis/deduplication.go

```diff
@@ -56,10 +81,25 @@ func combineCertOnDiskLocations(in *certgraphapi.CertKeyPair, rhs []certgraphapi
 			}
 		}
 		if !found {
+			// Store key to be merged into Cert
+			if curr.Cert.Path == "" {
```

Comment:
```suggestion
			if len(curr.Cert.Path) == 0{
```

for kube standard.

Still reading to figure out why we assign an empty path to `keyLocation`.  Possible logic inversion?

Reply:
This iterates over all found certs and keys in `rhs` and produces a new keypair. It would be a list of cert/key pairs, or a key or a cert separately. This is why keys (or certs) with empty path are not ignored - otherwise PEM certkey pairs would be missing (these are treated as certificates at the moment).